### ISSUE TITLE
Add extends Controller in stub only if it exists

### ISSUE THREAD

**Issue by felixdorn** [laravel-idea/plugin#325]
Hey!

The `Create Controller` action generates a controller that looks more or less like this :

```php
<?php

namespace App\Http\Controllers;

class SomeController extends Controller
{
    public function index()
    {
        //
    }
}

```

It would be awesome if the `extends Controller` part was added only if the class actually exists, I personally don't use it (and delete it!) and I pretty sure I'm not the only one who does that.

**Comment by adelf** [comment]
Hi, Felix.
You can configure base class in the settings: https://laravel-idea.com/docs/generation#configuration
However, you’re right. I’ll add some check about base class existence. Thanks!

**Comment by felixdorn** [comment]
Wow, should've looked into that more deeply, didn't even cross my mind to look for a setting.

Feel free to close the issue, I didn't close it myself in case you use them as TODOs.

Thanks for the great product :smile: !

**Comment by adelf** [comment]
Fixed in the 4.4 version. I don't make "extends" if the base class doesn't exist.
Full list of changes - https://laravel-idea.com/blog/2021/4-4-release
